### PR TITLE
Adding webpage content

### PR DESCRIPTION
**Pull request by jaimevillegas**
I have added an html file, a css file and some others that you can see below, in order to have a working web page showing some information.

Changes Made:
- Added index.html
- Added style.css
- Edited .gitignore file to ignore node_modules and test.md
- Added README.md file

Thank you a lot for taking the time to review my code and suggest any changes you consider.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (osamaashraf6) [inline README.md]: - [x] Great job 👌at updating the README file, but I see that the MIT LICENCE file is broken☺️, so kindly try to fix the LICENCE file by adding it from the GitHub💯

File: .gitignore [hunk 2]
```diff
@@ -1 +1,2 @@
 node_modules
+test.md
```
Comment (osamaashraf6) [inline .gitignore:2]: - [ ] You did well 👌at implementing the requirements of the project💯, but I see that you put the ``` test.md ``` file in the gitignore, but you didn't add the file to the directory, so please😊 put the test.md file in the directory of the project and put this sentence which inside the requirements.

**Review by osamaashraf6 — CHANGES_REQUESTED** [review]
Hi @jaimevillegas 👋, 

Good job so far! 👏🥳
You have done well reaching this point of the milestone, but there are some issues that you still need to work on to go to the next project but you are almost there! 

you-are-near

## Highlights ✔️
- [x] Good commit messages.
- [x] No linter errors. 💯
- [x] Good usage in correcting GitHub flow


## Required Changes ♻️

- [x] You did well 🎈🪄on the project, but I see that you didn't write a pull request for this project, so please😃 add a Pull Request which should have a descriptive title telling you what changes are inside and Pull Request should have a summary describing introduced changes in more detail.💯

- [x] Good job👍with the project you are almost there you finish the project don't worry 🥳your code is good, but there are some issues to be fixed, so please check the comments under the review to know what I am talking about 😊

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by jaimevillegas** [comment]
Hello @osamaashraf6 !

Thank you a lot for reviewing my code. I have worked on the tasks that you left me, and I think it is all resolved now.

If there are some other thing to do, don't hesitate to tell me! 

Best regards!

**Review by christianbiring1 — APPROVED** [review]
Hi @jaimevillegas , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 

## Highlights
- [ ] lInters checks are green :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**